### ISSUE TITLE
Finding WPF panel in WinAppDriver

### ISSUE THREAD

**Issue by khoachung89** [microsoft/WinAppDriver#818]
In my WPF window, i have  panel (stack panel, grid panel...)
In Inspector i can see them 

![cannotbefound](https://user-images.githubusercontent.com/4082657/63333564-dc7f0000-c363-11e9-9adf-78e662c3f2f8.jpg)

However i cannot select them with WinAppDriver (by accessibility, by class name....)

![cannotbefound3](https://user-images.githubusercontent.com/4082657/63333561-dc7f0000-c363-11e9-9135-0d069cca9723.jpg)
![cannotbefound2](https://user-images.githubusercontent.com/4082657/63333563-dc7f0000-c363-11e9-93d7-74bef3ebebb7.jpg)

The other control like button or textbox can still be found normally.
Is there anyway i can get the panel of WPF from WinAppDriver?

**UPDATE:**
Looks like not only panel, but any control that doesn't have user interaction (a line for example) cannot be selected too. Is it a designation or a bug? I really need the panel coordination and size, without selecting it, how can i get it?

**Comment by khoachung89** [comment]
Close this issue because i can find them via XPath.

**Comment by Vasanthi30** [comment]
Hey, i am also facing the same issue, could you please provide the XPath what u have written to access that panel.

**Comment by khoachung89** [comment]
@Vasanthi30 
You may have a look at your session's PageSource before write XPath query because it's much easier to see than using Inspector.
My example case:
`WindowSession.FindElementByXPath("/Window/Pane")` will return exactly the pane that I'm selecting in my question's screenshot.
or:
`WindowSession.FindElementByXPath("/Window/Pane[@FrameworkId=\"WPF\"]").GetAttribute("ClassName")` will return `"ScrollViewer"`

**Comment by Vasanthi30** [comment]
Many Thanks @khoachung89 

Are you creating session similar to the below lines of code?
 DesiredCapabilities appCapabilities = new DesiredCapabilities();
                appCapabilities.SetCapability("app", _applicationPath);
                appCapabilities.SetCapability("deviceName", "WindowsPC");
               Session = new WindowsDriver<WindowsElement>(new Uri(WindowsApplicationDriverUrl), appCapabilities);

**Comment by khoachung89** [comment]
Yes, something like that.
